hq/app/model/models.scala

```diff
@@ -135,3 +135,23 @@ case class MachineUser  (
   reportStatus: ReportStatus,
   lastActivityDay : Option[Long]
 )
+
+case class Token(value: String) extends AnyVal
+
+case class Organisation(value: String) extends AnyVal
+
+case class SnykOrg(name: String, id: String)
+
+case class SnykProject(name: String, id: String)
+
+case class SnykIssue(title: String, id: String, severity: String)
+
+case class SnykProjectIssues(name: String, id: String, ok: Boolean, vulnerabilities: List[SnykIssue])  {
+  def withName(name: String) = new SnykProjectIssues(name, this.id, this.ok, this.vulnerabilities)
```

Comment:
These `with` methods are mixing the definition of the data structure with the program's logic. It's only used once and `copy` exists, why not just do that where you need it?